content/en/docs/instrumentation/php/automatic.md

```diff
@@ -43,35 +42,43 @@ $demo->run();
 
 Here, we provide `pre` and `post` functions, which are executed before and after
 `DemoClass::run`. The `pre` function starts and activates a span, and the `post`
-function ends it.
+function ends it. If an exception was thrown by `DemoClass::run()`, the `post`
+function will record it (without affecting exception propagation).
 
 ## Setup
 
-1. Install the extension via [pickle](https://github.com/FriendsOfPHP/pickle) or
+1. Install the extension via pecl,
+   [pickle](https://github.com/FriendsOfPHP/pickle) or
    [php-extension-installer](https://github.com/mlocati/docker-php-extension-installer)
-   (docker specific):
+   (docker specific).
 
-   - **pickle** can be used to install extensions that are available via
-     <http://pecl.php.net>, however that's not the case for
-     opentelemetry-php-instrumentation yet, so the only way for it is to install
-     directly from source code. The following command line shows you how to do
-     that using a specific version of the extension (1.0.0beta2 in this case):
-
-     Installing from source requires proper development environment and few
-     dependencies:
+   Installing from source requires proper development environment and few
+   dependencies:
 
      <!-- prettier-ignore-start -->
 
-     {{< tabpane lang=shell persistLang=false >}}
+   {{< tabpane lang=shell persistLang=false >}}
 
-     {{< tab "Linux (apt)" >}}sudo apt-get install gcc make autoconf{{< /tab >}}
+   {{< tab "Linux (apt)" >}}sudo apt-get install gcc make autoconf{{< /tab >}}
 
-     {{< tab "MacOS (homebrew)" >}}brew install gcc make autoconf{{< /tab >}}
+   {{< tab "MacOS (homebrew)" >}}brew install gcc make autoconf{{< /tab >}}
 
-     {{< /tabpane >}}
+   {{< /tabpane >}}
      <!-- prettier-ignore-end -->
 
-     With your environment setup you can install the extension:
+   With your environment setup you can install the extension in a variety of
+   ways:
```

Comment:
Maybe it would make sense to put the three methods/options inside tabbed panes?

If not, at a minimum, the indentation of one `pecl` command block needs to be fixed -- see the next comment.

Reply:
I've reworked that section, using tabs.